spark/src/main/scala/org/apache/comet/serde/QueryPlanSerde.scala

```diff
@@ -186,6 +186,13 @@ object QueryPlanSerde extends Logging with ShimQueryPlanSerde {
     }
   }
 
+  private def bitwiseAggTypeSupported(dt: DataType): Boolean = {
+    dt match {
+      case _: IntegerType => true
```

Comment:
I think Spark supports all `Integral` type as input type which includes other types such as long and short. Maybe we can add them later.

Reply:
Added all the other `IntegralType`